devtron-labs/dashboard
# Fix link to git access credential in GitOps config

# Description

Fix the link to creating git access credential in the GitOps config page to an accurate and latest link.
A very minor change but the new link will now navigate to the right section of the docs instead of the start of the page.

Fixes # (issue)

## Type of change


# How Has This Been Tested?

- Run the dashboard and visit the GitOps configuration page under the Global Settings (/dashboard/global-config/gitops)
- Click on the link for `Check permissions required for PAT`
- Verify that the documentation opened navigates to the right section

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note). **No related issues since it is a minor change**
* [x] Does this PR require documentation updates? **No**
* [x] I've updated documentation as required by this PR. **N/A**
